databases/week2/creatingDatabase/week2HWscript.sql

```diff
@@ -0,0 +1,102 @@
+CREATE TABLE `customers` (
+  `customerID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `firstName` varchar(255) NOT NULL,
+  `lastName` varchar(255) NOT NULL,
+  `email` varchar(255) NOT NULL UNIQUE,
+  `phone` varchar(255) NOT NULL,
```

Comment:
Nothing wrong, but why you want to assign 255 char for fisrtname,lastname, email or phone. It's not logical to assign this much as fx. a phone number can max be 20 characters... Would say `varchar(50)`

Reply:
yeah you are right actually i just wanted be sure :)